packages/database/src/migrations/20210325001804-AddCovidVaccineTrackingDashboardGroups-modifies-data.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+import { insertObject, arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+const dashboardGroups = [
+  // Covid Fiji
+  {
+    organisationLevel: 'Country',
+    organisationUnitCode: 'FJ',
+    name: 'COVID-19 Fiji',
+    code: 'FJ_Covid_Fiji_Country_COVID-19',
+    projectCodes: `{}`,
```

Comment:
Do we need to specify the `projectCodes`?

Reply:
YES! yes we do!